### ISSUE TITLE
Add a new field of Per Person Ventilation Rate Mode to AirTerminal:SingleDuct:Uncontrolled and AirTerminal:SingleDuct:Mixer

### ISSUE THREAD

**Issue by lgu1234** [NREL/EnergyPlus#6241]
Issue overview
--------------
The new field will give users a choice to calculate ventilation rate based on either design occupancy and current occupancy.

### Details
Some additional details for this issue (if relevant):
 - Platform (Operating system, version)
 - Version of EnergyPlus (if using an intermediate build, include SHA)
 - Unmethours link or helpdesk ticket number

### Checklist
Add to this list or remove from it as applicable.  This is a simple templated set of guidelines.
 - [ ] Defect file added (list location of defect file here)
 - [ ] Ticket added to Pivotal for defect (development team task)
 - [ ] Pull request created (the pull request will have additional tasks related to reviewing changes that fix this defect)

**Comment by mjwitte** [comment]
@Myoldmopar This is one more item with IDD changes that@lgu1234 was planning to get into v8.8.  I had asked for this to be a separate PR after #6223 drops in.

**Comment by Myoldmopar** [comment]
@lgu1234 if you want to go ahead and get started on this, just create a new branch based on the #6223 branch and work on from that point.  I suspect #6223 will be merged into develop today, so once that gets merged in you'll be good to go.

**Comment by lgu1234** [comment]
@Myoldmopar I can work on my local computer first.

**Comment by lgu1234** [comment]
@mjwitte @Myoldmopar I will check in within an hour.

**Comment by rraustad** [comment]
closed via #6252.
